components/automate-deployment/pkg/converge/converger.go

```diff
@@ -1,12 +1,16 @@
 package converge
```

Comment:
All the state machine changes are in this file, you probably need to expand it in github.